cordapp/src/test/kotlin/com/template/FlowTests.kt

```diff
@@ -1,31 +1,27 @@
 package com.template
 
 import net.corda.testing.node.MockNetwork
-import net.corda.testing.node.StartedMockNode
 import org.junit.After
 import org.junit.Before
 import org.junit.Test
 
 class FlowTests {
-    lateinit var network: MockNetwork
-    lateinit var a: StartedMockNode
-    lateinit var b: StartedMockNode
 
-    @Before
-    fun setup() {
-        network = MockNetwork(listOf("com.template"))
-        a = network.createNode()
-        b = network.createNode()
+    private val network = MockNetwork(listOf("com.template"))
+    private val a = network.createNode()
+    private val b = network.createNode()
```

Comment:
Won't this cause the tests to have shared state once we start adding more?

Reply:
No, because JUnit creates a new instance of the test class before invoking each @Test method.